### ISSUE TITLE
Secure connection attempt prints out gibberish if trying to connect locally instead of to a remote system

### ISSUE THREAD

**Issue by MikeFultonDev** [ZOSOpenTools/c3270port#14]
Specifically, both:

`c3270 Y:L:aqft.pok.ibm.com:992`

and

`c3270 -noverifycert L:aqft.pok.ibm.com:992`

will connect locally on AQFT.POK.IBM.COM but:

`c3270 Y:L:127.0.0.1:992`

produces:

![image](https://github.com/ZOSOpenTools/c3270port/assets/61100689/df9fa402-9120-424d-b96f-4c465fb8f2e7)

**Comment by MikeFultonDev** [comment]
The symptoms are different now - likely due to changes in the TLS configuration of the 2 systems I am using.
Now I get gibberish on the screen if I try to connect to a localhost - whether it is with a fully qualified name or with 127.0.0.1, but I _can_ connect to another z/OS system from z/OS with c3270, e.g.

```
c3270 Y:L:aqft.pok.ibm.com:992
```
connects properly from zoscan2b and:

```
c3270 Y:L:zoscan2b.pok.stglabs.ibm.com:992
```
connects properly from aqft.
But I can't connect to aqft from aqft or zoscan2b from zoscan2b. So that 'local' path has issues of some form.

**Comment by MikeFultonDev** [comment]
fixed in PR #21 with fixes in the mainline (timing bug exposed on z/OS)
